knime_extension/src/util/knime_utils.py

```diff
@@ -550,3 +550,25 @@ def Turn_all_NA_column_as_str(gdf) -> None:
         gdf[Nacol] = gdf[Nacol].astype(str)
     gdf = gdf.reset_index(drop=True)
     return gdf
+
+def re_order_weight_rows(gdf,adjust_list,id_col):
```

Comment:
Maybe move this method and the parameter definition (similar to this https://github.com/spatial-data-lab/knime-geospatial-extension/blob/main/knime_extension/src/util/knime_utils.py#L95) into its own file in the util folder?